src/x509.c

```diff
@@ -5444,63 +5483,62 @@ static int X509PrintSerial(WOLFSSL_BIO* bio, WOLFSSL_X509* x509, int indent)
     unsigned char serial[32];
     int  sz = sizeof(serial);
     char scratch[MAX_WIDTH];
+    int scratch_len;
 
     XMEMSET(serial, 0, sz);
     if (wolfSSL_X509_get_serial_number(x509, serial, &sz) == WOLFSSL_SUCCESS) {
-
-        XSNPRINTF(scratch, MAX_WIDTH, "%*s%s", indent, "",
-                "Serial Number:");
-        if (wolfSSL_BIO_write(bio, scratch, (int)XSTRLEN(scratch)) <= 0) {
+        if ((scratch_len = XSNPRINTF(scratch, MAX_WIDTH, "%*sSerial Number:",
+                                     indent, "")) >= MAX_WIDTH)
+        {
```

Comment:
lets keep the { on the same line with the if statement

Reply:
other spots like this too later on

Reply:
yeah that one didn't serve any purpose to visual clarity so I tweaked it as you describe.  but the others are important for visual clarity.  compare:
```
            if ((scratchLen = XSNPRINTF(
                     scratch, MAX_WIDTH, " %d (0x%x)\n", serial[0], serial[0]))
                >= MAX_WIDTH)
            {
                WOLFSSL_MSG("buffer overrun");
                return WOLFSSL_FAILURE;
            }
```
vs
```
            if ((scratchLen = XSNPRINTF(
                     scratch, MAX_WIDTH, " %d (0x%x)\n", serial[0], serial[0]))
                >= MAX_WIDTH) {
                WOLFSSL_MSG("buffer overrun");
                return WOLFSSL_FAILURE;
            }
```
it's happening because we use 4 space indentation, and "if (" is also 4 characters long.

the lines have to be wrapped because we have an 80 character line length rule.

so, without manually using ad hoc indentation that will be destroyed by any subsequent auto-indentation, this is the only way to avoid the test clause and the true clause running together visually.

when the cost is small or zero, avoiding confusion is a very good reason to do it!

by the way, my editor's electric keys do indeed destroy the large amount of ad hoc indentation in our code.  I have to go to a significant amount of trouble to suppress this behavior on a case-by-case basis.  it is very distracting and annoying.